include/nestegg/nestegg.h

```diff
@@ -439,6 +452,18 @@ int nestegg_packet_encryption(nestegg_packet * packet);
 int nestegg_packet_iv(nestegg_packet * packet, unsigned char const ** iv,
                       size_t * length);
 
+/** Query the packet for offsets.
+@param packet            Packet initialized by #nestegg_read_packet.
+@param partition_offsets Storage for queried offsets.
+@param num_offsets       Length of returned offsets, may be 0.
+The data is owned by the #nestegg_packet packet.
+@retval  0 Success.
+@retval -1 Error.
+*/
+int nestegg_packet_offsets(nestegg_packet * packet,
+                           uint32_t ** partition_offsets,
```

Comment:
This should be `const` to indicate to the caller that the data is read-only (which also provides a weak signal that the caller is not required to free the data).